app/models/user.rb

```diff
@@ -82,6 +85,14 @@ def can_create_game?
     owned_games.size <= 10
   end
 
+  def notify_facebook_friends
```

Comment:
I think this method should be extracted into a job to be run on background otherwise the login/sign up will take a lot of time...